backend/api/Areas/Property/Models/Parcel/SubdivisionParcelModel.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using Model = Pims.Api.Models;
+
+namespace Pims.Api.Areas.Property.Models.Parcel
+{
+    public class SubdivisionParcelModel : Model.BaseModel
```

Comment:
How is this different from `ParcelSubdivisionModel `?

Reply:
I could reuse the same model, but I'm trying to convey the fact that the service returns a divided parcel with a list of subdivisions or a subdivision parcel with a list of divided parent parcels.

For now there is no difference between a subdivision parcel model and a parent divided parcel, but that may not always be the case.